### PR TITLE
[EventHubs] Eventhubs runtime metric

### PR DESCRIPTION
**Pull request by yunhaoling**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py [hunk 1]
```diff
@@ -179,12 +191,31 @@ def _create_handler(self):
             event_data = EventData._from_message(message)  # pylint:disable=protected-access
             self._offset = EventPosition(event_data.offset)
             data_batch.append(event_data)
+
+        if self._track_last_enqueued_event_info and len(data_batch) > 0:
```
Comment (bryevdv) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py]: Suggest: 

```
if data_batch and self._track_last_enqueued_event_info:
```

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py [hunk 2]
```diff
@@ -122,6 +128,11 @@ def _create_handler(self):
         source = Source(self._source)
         if self._offset is not None:
             source.set_filter(self._offset._selector())  # pylint:disable=protected-access
+
+        desired_capabilities = utils.data_factory(
+            types.AMQPArray([types.AMQPSymbol(self._receiver_runtime_metric_symbol)]))\
+            if self._track_last_enqueued_event_info else None
```
Comment (bryevdv) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py]: FWIW I think ternary ops are best when things are short and fit on a single line.  Only a suggestion but something like this is more approachable (to me)
```
desired_capabilities = None
if self._track_last_enqueued_event_info: 
    symbol = self._receiver_runtime_metric_symbol
    desired_capabilities = data_factory(AMQPArray([AMQPSymbol(symbol)]))
```

File: sdk/eventhub/azure-eventhubs/azure/eventhub/consumer.py [hunk 3]
```diff
@@ -173,12 +185,31 @@ def _receive(self, timeout_time=None, max_batch_size=None, **kwargs):
             event_data = EventData._from_message(message)  # pylint:disable=protected-access
             self._offset = EventPosition(event_data.offset)
             data_batch.append(event_data)
+
+        if self._track_last_enqueued_event_info and len(data_batch) > 0:
```
Comment (bryevdv) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/consumer.py]: same comment as above

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py [hunk 4]
```diff
@@ -266,14 +266,16 @@ def create_consumer(
         owner_level = kwargs.get("owner_level")
         operation = kwargs.get("operation")
         prefetch = kwargs.get("prefetch") or self._config.prefetch
+        track_last_enqueued_event_info = kwargs.get("track_last_enqueued_event_info", False)
```
Comment (YijunXieMS) [inline sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py]: Add the docstring for this param

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py (hunk 1 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py]: thanks for the suggestion, updated

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py (hunk 2 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py]: good suggestion, code updated

File: sdk/eventhub/azure-eventhubs/azure/eventhub/consumer.py (hunk 3 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/consumer.py]: code updated

File: sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py (hunk 4 above)
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhubs/azure/eventhub/aio/client_async.py]: docstring has been updated for this param

**Pull request merged**